### PR TITLE
Add notification sound when new message is received

### PR DESCRIPTION
**Pull request by AngieCristina**
**Implementation notes**

- Added a new sound to the assets folder.
- Added a conditional statement to play the notification sound only if the sender is not the current user.

### CODE REVIEW

File: public/js/main.js [hunk 1]
```diff
@@ -149,6 +149,12 @@ function receiveComment(inputMessage) {
   channel.messages.push(newMessage);
   saveChannelStorage();
   renderComments(activeChannel);
+    
+  if(localStorage.getItem("currentUser") != newMessage.author) {
```
Comment (valevassallo) [inline public/js/main.js]: here aren't you comparing if that currentUser doesn't equal the message's author? When comparing use the double equal sign (==, or in this case !==)

Comment (MariethPerez) [reply public/js/main.js]: I think that in this line already is comparing the currentUser with the message's author. When you use !== is a strict comparing.

File: public/js/main.js [hunk 2]
```diff
@@ -149,6 +149,12 @@ function receiveComment(inputMessage) {
...
+    const sound = new Audio("../assets/notification-sound.mp3");
+    sound.play();
```
Comment (MariethPerez) [inline public/js/main.js]: Thanks for your contribution. :)

File: public/js/main.js (hunk 1 above)
Comment (MariethPerez) [reply public/js/main.js]: Here I put a link about comparison operator: https://developer.mozilla.org/es/docs/Web/JavaScript/Referencia/Operadores/Comparison_Operators

Comment (AngieCristina) [reply public/js/main.js]: Good catch! Thank you for your comments, both are correct .In fact what Marieth mentioned is correct that `!=` is making his work of comparing, let change into a *strict comparing*, I will send amended of this pull request

File: public/js/main.js (hunk 2 above)
Comment (CesarCachay) [reply public/js/main.js]: Great!  Thanks it was really good!

**Pull request merged**